src/hooks/useLink.tsx

```diff
@@ -0,0 +1,25 @@
+import type { KeyboardEvent, MouseEvent } from 'react'
+import { useNavigation, useView } from '@/hooks'
+import { handleLink } from '@/components/Link/lib/handleLink'
+
+export const useLink = (viewName: 'Editor' | 'PlanningOverview' | 'Planning') => {
```

Comment:
```suggestion
import { View } from 'what-ever-path'
export const useLink = (viewName: View) => {
```
Would this work? Then we don't need to change the types when/if we add views?

Reply:
Yes, that is a good idea! Otherwise this will be a headache in the future. I'll add this as a separate issue though,